src/App.tsx

```diff
@@ -15,36 +16,96 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  SORT_OFF = '',
+  SORT_ABC = 'abc',
+  SORT_BY_LENGTH = 'leng',
+}
+
+function getGoodsView(sortField: SortType, reverse: boolean) {
+  let prepearedGoods = [...goodsFromServer];
+
+  switch (sortField) {
+    case SortType.SORT_ABC:
+      prepearedGoods = prepearedGoods.sort((cur, prew) => {
+        return reverse ? prew.localeCompare(cur) : cur.localeCompare(prew);
+      });
+      break;
+    case SortType.SORT_BY_LENGTH:
+      prepearedGoods = reverse
+        ? prepearedGoods.sort((a, b) => a.length - b.length).reverse()
+        : prepearedGoods.sort((a, b) => a.length - b.length);
+      break;
+    default:
+      prepearedGoods = reverse
+        ? prepearedGoods.reverse()
+        : [...goodsFromServer];
```

Comment:
In the `default` case, when `reverse` is true, the code reverses `prepearedGoods`, which is initially a copy of `goodsFromServer`. However, if `reverse` is false, it resets `prepearedGoods` to a new copy of `goodsFromServer`. This logic might be inconsistent if the intention is to always reverse the original order when `reverse` is true. Consider ensuring that the logic consistently applies the reverse operation to the original list.